templates/zerver/archive/home.html

```diff
@@ -0,0 +1,15 @@
+{% if is_web_public and message_list %}
+<div class="message_table focused_table" id="zhome">
+    {% for msg in message_list %}
+    {{ msg|safe }}
+    {% endfor %}
+</div>
+{% elif is_web_public %}
+<p>
+    This topic does not exist.
```

Comment:
I think "No messages have been sent to this topic within the publicly archived stream #%(stream_name)s".